app/assets/stylesheets/components/claim_deadline.scss

```diff
@@ -0,0 +1,25 @@
+@import "govuk-frontend-rails";
+
+%claim-deadline {
+  text-transform: lowercase;
```

Comment:
Probably better to make sure we're actually using lowercase text so that copying it (and I guess screen readers?) don't have the raw text with uppercase letters.

Reply:
Yeah the text is actually lowercase but the `govuk-tag` class makes it appear uppercase. I've change our override to be `text-transform: none`

Reply:
Aha! Makes sense.